### PR TITLE
update to docs v3

### LINKED ISSUE #779: Docs v1 Update Checklist
# Module Update Checklist
----
Name of Module: Directories and File Paths
PR for module update: #[PR number here]

This is the checklist for bringing a module up to date with the latest major version of docs.md: v1

## Summary of changes

The biggest changes to docs for this version are extensive updates to the required front matter, including lots of new fields, and the introduction of shared macros to replace standardized sections.

The following sections provide a complete (hopefully!) list of potential changes to modules needed to bring them up to date with the new version of docs.md. 

## Front matter

* [ ] The front matter includes all required fields from the new templates. Some of these are new fields and some are not. Use the ["Your front matter checklist"](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/docs.md#your-front-matter-checklist) section of docs.md to get a customized list of the required front matter for a given module. 

Also watch specifically for the following:

* [ ] current_version_description is a short, present-tense text fragment without bullets, and gives a learner-centric quick overview of version changes from the learner perspective
* [ ] version_history contains 1-3 past versions with bulleted links to commit-specific documents, unless this is an initial version of a module, in which case version_history should say "No previous versions." For detailed instructions on how to fill out the version fields, see the [docs descriptions](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/docs.md#version_history).

The following should **NOT** be included in module front matter: 

- [ ] REMOVED: any macros defined in the shared macro files
- [ ] REMOVED: `link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css` and `link: https://storage.googleapis.com/chop-dbhi-arcus-education-website-assets/css/styles.css`
- [ ] REMOVED: `script: https://kit.fontawesome.com/83b2343bd4.js`
- [ ] REMOVED: `script:  https://code.jquery.com/jquery-3.6.0.slim.min.js`

## Macros

* For all modules:
    - [ ] Overview is created with the macro, not written
    - [ ] Feedback is created with the macro, not written

* For modules with interactive R, Python, or SQL:
    - [ ] Lesson Preparation is created with the relevant macro, not written

* For wrapper modules:
    - [ ] Lesson Preparation is created with the relevant macro, not written
    - [ ] There is a `print_resource` macro for each external resource just after the `lesson_prep_wrapper` macro

## Other changes

* [ ] Avoid linking to our own modules in the Additional Resources section at the end of each module
* [ ] Check that highlight boxes use the updated style

## Final check

These updates should not change the rendered module much, for the most part. 
To check, pull up the rendered version from before the update and your updated version and look for missing/changed/added content, or anything that's not rendering properly. 
For conveience, paste the liascript links here:

- Original: {paste liascript link for original module}
- Update: {paste liascript link for updated module}

When you're ready to finalize the updated module, do the following in the module front matter:

* [ ] Increment `version` for the module. If no meaningful changes were made to the content of the module as it renders for learners, this is a revision. If there were updates to the module content (e.g. in the Additional Resources section), then it's a minor update.
* [ ] Update `docs_version`

### LINKED ISSUE #780: Docs v2 Update Checklist
# Module Update Checklist
----
Name of Module: Directories and File Paths
PR for module update: #[PR number here]

This is the checklist for bringing a module up to date with the latest major version of docs.md: v2

## Summary of changes

The only change is that `coding_required` is now a mandatory front matter field. 
It must be either `true` or `false`. 
See [the `coding_required` page in docs](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/docs.md#38) for more details about this field. 

## Front matter

* [x] `coding_required` is either `true` or `false`

## Final check

When you're ready to finalize the updated module, do the following in the module front matter:

* [x] Increment `version` for the module. This is a revision. 
* [x] Update `docs_version` to 2.0.0

### LINKED ISSUE #781: Docs v3 Update Checklist
# Module Update Checklist
----
Name of Module: Directories and File Paths
PR for module update: #[PR number here]

This is the checklist for bringing a module up to date with the latest major version of docs.md: v3

## Summary of changes

An optional front matter field, `collection`, is now available. Check if this module should belong in one or more collections. To learn more, read the [description in docs](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/docs.md#collection).

There is also a new option, `text`, for `data_domain`.

Every module should have at least one of `data_domain`, `data_task` or `collection` populated with something.

## Front matter

* [x] If relevant, `collection` is listed, with one or more collections (separated by commas)
* [x] If relevant, add `text` as the data domain

## Final check

When you're ready to finalize the updated module, do the following in the module front matter:

* [x] Increment `version` for the module. This is a revision. 
* [x] Update `docs_version` to 3.0.0

### PR DESCRIPTION
**Pull request by franzenr**
resolves #779
resolves #780
resolves #781

brings this module up to alignment with v1-3 of docs

### CODE REVIEW

File: directories_and_file_paths/directories_and_file_paths.md [hunk 1]
```diff
@@ -26,35 +32,29 @@ After completion of this module, learners will be able to:
...
 
-**Estimated time to completion:** @estimated_time
+@end
 
-**Pre-requisites**: None.
+@depends_on_knowledge_available_in: 
 
-**Learning Objectives**:
+@end
+@version_history
```
Comment (leemc-data-ed) [inline directories_and_file_paths/directories_and_file_paths.md:45]: Version history is not rendering correctly in Liascript for some reason-- but I'm not immediately sure why? The section is visible but the versions and descriptions I see in the raw markdown are different in the rendered page (it looks like sample text). I checked in Chrome and Safari.

**Review by leemc-data-ed — CHANGES_REQUESTED** [review]
Just one issue I found with version history, otherwise looks good!

**Review by rosemm — APPROVED** [review]
LGTM

**Comment by rosemm** [comment]
Lol @leemc-data-ed I just realized you and I both had this PR in our jira tickets by mistake --- I thought you were just jumping in and being helpful 😂

**Comment by leemc-data-ed** [comment]
Haha well better than no one being assigned I guess!

File: directories_and_file_paths/directories_and_file_paths.md (hunk 1 above)
Comment (leemc-data-ed) [reply directories_and_file_paths/directories_and_file_paths.md:45]: Oooh amazing, thanks for figuring this out!

**Review by leemc-data-ed — APPROVED** [review]
LGTM!

**Pull request merged**